modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -10,9 +10,8 @@ class PowerOfAttorneyController < ClaimsApi::V1::Forms::Base
         include ClaimsApi::EndpointDeprecation
         include ClaimsApi::PoaVerification
 
-        before_action except: %i[schema] do
```

Comment:
schema got moved up to `ClaimsApi::V1::Forms::Base`, so this wasn't needed and forgotten to be removed, as is